app/src/main/java/com/gravatar/app/GravatarApplication.kt

```diff
@@ -12,7 +13,7 @@ class GravatarApplication : Application() {
 
         startKoin {
             androidContext(this@GravatarApplication)
-            modules(appModule)
+            modules(appModule, analyticsModule)
```

Comment:
❓ I'm curious, is there a difference between this and adding the `analyticsModule` in the `appModule` via `includes()`?

Reply:
hmm looks like the end outcome is the same. I would lean towards including it in the `appModule` given that the `:app` glues the whole app, meaning all its dependencies.